airflow/contrib/kubernetes/worker_configuration.py

```diff
@@ -226,6 +227,9 @@ def make_pod(self, namespace, worker_uuid, pod_id, dag_id, task_id, execution_da
         if gcp_sa_key:
             annotations['iam.cloud.google.com/service-account'] = gcp_sa_key
 
+        annotations['dag_id'] = dag_id
```

Comment:
Is it worth "namespacing" these annotations to something like `org.apache.airflow.dag_id`?

(I don't know if that's recommended or needed in kube-land)

Reply:
Yes you're likely right.  From the docs, it does mention that automated processes must specify a prefix, so with that I think `org.apache.airflow/dag_id` and `org.apache.airflow/task_id` would probably be preferential.